src/jaxsim/mujoco/__main__.py

```diff
@@ -0,0 +1,191 @@
+        "--description",
+        required=True,
+        metavar="INPUT_FILE",
+        type=pathlib.Path,
+        help="Path to the URDF or SDF file.",
+    )
+
+    parser.add_argument(
+        "-m",
+        "--model-name",
+        metavar="NAME",
+        type=str,
+        default=None,
+        help="The target model of a SDF description if multiple models exists.",
+    )
+
+    parser.add_argument(
+        "-e",
+        "--export",
+        metavar="MJCF_FILE",
+        type=pathlib.Path,
+        default=None,
+        help="Path to the exported MJCF file.",
+    )
+
+    parser.add_argument(
+        "-f",
+        "--force",
+        action="store_true",
+        default=False,
+        help="Override the output MJCF file if it already exists (default: %(default)s).",
+    )
+
+    parser.add_argument(
+        "-p",
+        "--print",
+        action="store_true",
+        default=False,
+        help="Print in the stdout the exported MJCF string (default: %(default)s).",
+    )
+
+    parser.add_argument(
+        "-v",
+        "--visualize",
+        action="store_true",
+        default=False,
+        help="Visualize the description in the Mujoco viewer (default: %(default)s).",
+    )
+
+    parser.add_argument(
+        "-b",
+        "--base-position",
+        metavar=("x", "y", "z"),
+        nargs=3,
+        type=float,
+        default=None,
+        help="Override the base position (supports only floating-base models).",
+    )
+
+    parser.add_argument(
+        "-q",
+        "--base-quaternion",
+        metavar=("w", "x", "y", "z"),
+        nargs=4,
+        type=float,
+        default=None,
+        help="Override the base quaternion (supports only floating-base models).",
+    )
+
+    args = parser.parse_args()
+
+    # ==================
+    # Validate arguments
+    # ==================
+
+    # Expand the path of the URDF/SDF file if not absolute.
+    if args.description is not None:
+        args.description = (
+            (
+                args.description
+                if args.description.is_absolute()
+                else pathlib.Path.cwd() / args.description
+            )
+            .expanduser()
+            .absolute()
+        )
+
+        if not pathlib.Path(args.description).is_file():
+            msg = f"The URDF/SDF file '{args.description}' does not exist."
+            parser.error(msg)
+            exit(1)
+
+    # Expand the path of the output MJCF file if not absolute.
+    if args.export is not None:
+        args.export = (
+            (
+                args.export
+                if args.export.is_absolute()
+                else pathlib.Path.cwd() / args.export
+            )
+            .expanduser()
+            .absolute()
+        )
+
+        if pathlib.Path(args.export).is_file() and not args.force:
+            msg = "The output file '{}' already exists, use '--force' to override."
+            parser.error(msg.format(args.export))
+            exit(1)
```

Comment:
```suggestion
            sys.exit(1)
```
Contrary to 'exit()', 'sys.exit' does not rely on the site module being available

Reply:
Addressed in https://github.com/ami-iit/jaxsim/pull/83/commits/4dcd4a760ac2d9afe7d6a74fea385f2447ebb07f